odata-parser.pegjs

```diff
@@ -671,11 +676,6 @@ Apostrophe =
 	/	'%27'
 		{ return '\'' }
 
-Dollar =
```

Comment:
@Page- you can use `Dollar` in `QueryOption` and then rewrite this as `Dollar "Dollar before query option" =`. This will cause error messages to read `Expected Dollar before query option but instead ...`. Basically you can add a custom message wrapped in "" after a rule to customise the error